arrow/arrow_test.go

```diff
@@ -0,0 +1,130 @@
+package arrow_test
+
+import (
+	"encoding/hex"
+	"log"
+	"os"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"github.com/tarantool/go-tarantool/v2"
+	"github.com/tarantool/go-tarantool/v2/arrow"
+	"github.com/tarantool/go-tarantool/v2/test_helpers"
+)
+
+var isArrowSupported = false
+
+var server = "127.0.0.1:3013"
+var dialer = tarantool.NetDialer{
+	Address:  server,
+	User:     "test",
+	Password: "test",
+}
+var space = "testArrow"
+
+var opts = tarantool.Opts{
+	Timeout: 5 * time.Second,
+}
+
+func skipIfArrowUnsupported(t *testing.T) {
+	t.Helper()
+	if !isArrowSupported {
+		t.Skip("Skipping test for Tarantool without Arrow support in msgpack")
+	}
+}
+
+// TestInsert uses Arrow sequence from Tarantool's test .
+// nolint:lll
+// See: https://github.com/tarantool/tarantool/blob/master/test/box-luatest/gh_10508_iproto_insert_arrow_test.lua
```

Comment:
https://github.com/tarantool/go-tarantool/blob/592db69eed8649b82ce432b930c27daeee98c52f/.golangci.yaml#L30
```suggestion
// See:
// https://github.com/tarantool/tarantool/blob/master/test/box-luatest/gh_10508_iproto_insert_arrow_test.lua
```

Reply:
Update rule at `.golangci.yaml` in separate commit.